cpp/src/tutorial1_display.cpp

```diff
@@ -133,7 +98,6 @@ int main(int argc, char *argv[])
       int ret = video<uint16_t>(width, height, pixelformat, num_device);
     }
 
-
   }catch(std::exception& e){
```

Comment:
It should catch ion::Error additionally to std::exception.

Reply:
fixed in 87d09cf